gradle.properties

```diff
@@ -21,3 +21,4 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
**question:** why `biuldFeature {buildConfig : true}` is moved from from gradle files to gradle.properties? Is there any benefit of the new approach?

Reply:
it does not have to be defined per each module I think it is something we want to have anyways ?